config.cmake

```diff
@@ -248,6 +248,7 @@ config_string(
     UNQUOTE
 )
 config_option(KernelFastpath FASTPATH "Enable IPC fastpath" DEFAULT ON)
+config_option(KernelSignalFastpath SIGNAL_FASTPATH "Enable notification signal fastpath" DEFAULT OFF)
```

Comment:
If this is only tested with MCS, the config option needs to depend on MCS, otherwise people will think it is safe to use.

Reply:
Sure, done.